hack/jenkins/job-configs/kubernetes-jenkins/kubernetes-e2e.yaml

```diff
@@ -518,7 +524,8 @@
     name: kubernetes-e2e-gce-enormous-cluster
     test-owner: 'gmarek'
     emails: 'gmarek@google.com'
-    cron-string: 'H 8 * * *'
+    # Run only on Sunday at 8 am MTV time.
+    cron-string: 'H 8 * * 0'
```

Comment:
Like 2 more weeks - we have just stabilized it and I would like to confirm that it works.